packages/tracing-internal/src/browser/browserTracingIntegration.ts

```diff
@@ -564,6 +574,9 @@ function registerInpInteractionListener(
               routeName,
               duration,
               parentContext,
+              user,
+              activeTransaction,
+              replay,
```

Comment:
IMO we should pass in the id's here instead of holding references to replay integration/active transaction

Reply:
I should be able to update this for replay ID.

For profiles, the profiling integration only attaches the profile id to the active transaction when the transaction finishes. So we don't immediately have access to a profile id when an interaction occurs. Keeping a reference to the transaction seemed like the easiest way to grab the profile id when we're ready to send an INP. Not sure if there is a better solution, but I can think about it if this is a blocker

Reply:
ahh I see, yeah let's hold the transaction reference then, np